packages/grid/x-data-grid/src/tests/pagination.DataGrid.test.tsx

```diff
@@ -46,120 +46,158 @@ describe('<DataGrid /> - Pagination', () => {
     });
 
     it('should display the rows of page given in props', () => {
-      render(<BaselineTestCase page={1} pageSize={1} rowsPerPageOptions={[1]} />);
+      render(<BaselineTestCase paginationModel={{ page: 1, pageSize: 1 }} pageSizeOptions={[1]} />);
       expect(getColumnValues(0)).to.deep.equal(['1']);
     });
 
-    it('should not call onPageChange on initialisation or on page prop change', () => {
-      const onPageChange = spy();
+    it('should not call onPaginationModelChange on initialisation', () => {
+      const onPaginationModelChange = spy();
 
-      const { setProps } = render(
+      render(
         <BaselineTestCase
-          page={0}
-          pageSize={1}
-          onPageChange={onPageChange}
-          rowsPerPageOptions={[1]}
+          paginationModel={{ page: 0, pageSize: 1 }}
+          onPaginationModelChange={onPaginationModelChange}
+          pageSizeOptions={[1]}
         />,
       );
 
-      expect(onPageChange.callCount).to.equal(0);
-      setProps({ page: 2 });
-
-      expect(onPageChange.callCount).to.equal(0);
+      expect(onPaginationModelChange.callCount).to.equal(0);
     });
 
-    it('should allow to update the page from the outside', () => {
+    it('should allow to update the paginationModel from the outside', () => {
       const { setProps } = render(
-        <BaselineTestCase pageSize={1} page={0} rowsPerPageOptions={[1]} />,
+        <BaselineTestCase paginationModel={{ page: 0, pageSize: 1 }} pageSizeOptions={[1]} />,
       );
       expect(getColumnValues(0)).to.deep.equal(['0']);
-      setProps({ page: 1 });
+      setProps({ paginationModel: { page: 1, pageSize: 1 } });
       expect(getColumnValues(0)).to.deep.equal(['1']);
     });
 
-    it('should apply new page when clicking on next / previous button and onPageChange is not defined and page is not controlled', () => {
-      render(<BaselineTestCase pageSize={1} rowsPerPageOptions={[1]} />);
+    it('should not apply new page when clicking on next / previous button and onPaginationModelChange is not defined and page is not controlled', () => {
```

Comment:
Should we mention this behavior change in the migration guide?

Reply:
I think the title is confusing, this should be the text and then it should be the default behaviour.
```suggestion
    it('should not apply new page when clicking on next / previous button and onPaginationModelChange is not defined and paginationModel is controlled', () => {
```

Reply:
So the behavior hasn't changed, right?

Reply:
Yes, the same behavior that was previously for page or pageSize now works for paginationModel

Reply:
Great!